client/jetpack-onboarding/steps/woocommerce.jsx

```diff
@@ -43,16 +50,28 @@ class JetpackOnboardingWoocommerceStep extends React.PureComponent {
 				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
 
 				<div className="steps__button-group">
-					<Button href={ forwardUrl } onClick={ this.handleWooCommerceInstallation } primary>
-						{ translate( 'Yes, I am' ) }
-					</Button>
-					<Button href={ forwardUrl }>{ translate( 'Not right now' ) }</Button>
+					{ isPending || isCompleted ? (
+						<Button disabled>
+							{ isPending ? translate( 'Installing…' ) : translate( 'Installed' ) }
+						</Button>
+					) : (
+						<div>
+							<Button href={ forwardUrl } onClick={ this.handleWooCommerceInstallation } primary>
+								{ translate( 'Yes, I am' ) }
+							</Button>
+							<Button href={ forwardUrl }>{ translate( 'Not right now' ) }</Button>
+						</div>
+					) }
```

Comment:
That's a lot of tertiaries 😅

Okay to keep them, I guess. They are at least nicely arranged (disabled button for both pending and completed states, just with different labels, etc).